openstack/networking/v2/extensions/vpnaas/ipsecpolicies/testing/requests_test.go

```diff
@@ -166,3 +166,80 @@ func TestDelete(t *testing.T) {
 	res := ipsecpolicies.Delete(fake.ServiceClient(), "5c561d9d-eaea-45f6-ae3e-08d1a7080828")
 	th.AssertNoErr(t, res.Err)
 }
+
+func TestUpdate(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+
+	th.Mux.HandleFunc("/v2.0/vpn/ipsecpolicies/5c561d9d-eaea-45f6-ae3e-08d1a7080828", func(w http.ResponseWriter, r *http.Request) {
+		th.TestMethod(t, r, "PUT")
+		th.TestHeader(t, r, "X-Auth-Token", fake.TokenID)
+		th.TestHeader(t, r, "Content-Type", "application/json")
+		th.TestHeader(t, r, "Accept", "application/json")
+		th.TestJSONRequest(t, r, `
+	{
+		"ipsecpolicy":{
+	"name": "updatedname",
+	"description": "updated policy",
+	"lifetime": {
+	"value": 7000
+	}
+	}
+	}
```

Comment:
It looks like the formatting here has gone awry. Normally some mismatched indentation is OK but let's fix this block up.